### ISSUE TITLE
Extreme CPU usage

### ISSUE THREAD

**Issue by kevwil** [docker/docker#26750]
**Description**

My laptop fan sounds like an F-16 and the Activity Monitor shows "com.docker.hyperkit" process using 400% CPU usage. I am unable to investigate any further info because all docker commands time out.

**Steps to reproduce the issue:**
1. Install Docker for Mac
2. Wait

**Describe the results you received:**

See description.

**Describe the results you expected:**

Laptop not melting, docker able to build images without timing out, etc. ;-)

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `docker version`:**

times out

**Output of `docker info`:**

times out

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Output of Docker diagnostics window:

```Docker for Mac: version: 1.12.1 (2d5b4d9)
OS X: version 10.10.5 (build: 14F1912)
logs: /tmp/5D1D1D4F-73E4-4FAC-B4F8-3B6D33B52BB6/20160920-124937.tar.gz
failure: docker ps failed: Failure("docker ps: timeout after 10.00s")
[ERROR]  docker-cli
         docker ps failed
[OK]     virtualization kern.hv_support
[OK]     menubar
[OK]     moby-syslog
[OK]     dns
[OK]     disk
[OK]     system
[OK]     app
[OK]     osxfs
[OK]     virtualization VT-X
[OK]     db
[OK]     slirp
[OK]     logs
[OK]     env
[OK]     vmnetd
[OK]     moby-console
[OK]     moby
[OK]     driver.amd64-linux
Failure: Could not upload diagnostic data to remote server (docker-diagnose exit code is 1)```

Log file referenced in diagnostics output:
[20160920-124937.tar.gz](https://github.com/docker/docker/files/483414/20160920-124937.tar.gz)

**Comment by thaJeztah** [comment]
This may be a known issue, where in some cases docker starts to use a lot of CPU after the system wakes from (deep)sleep. I don't have the issue a hand, but quitting the docker app (from the menu icon), and starting it again should resolve it.

I don't think there's a clear cause found yet, but hope this helps.

/cc @justincormack who may know the issue its tracked in

**Comment by kevwil** [comment]
Thanks @thaJeztah for the update. I also just noticed there's a separate issue tracker for Docker 4 Mac issues, should I post there instead?

If there is a known issue with Docker 4 Mac and CPU usage after waking from sleep, this is quite likely related. I came back from lunch and my laptop started going crazy after logging back in. I dropped everything I was juggling at the time and rebooted, twice, not knowing the Docker menu icon could correctly restart this runaway process.
